src/main/java/com/google/sps/data/Reaction.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/** 
+ * Reactions a user can add to a post 
+ */
+public enum Reaction {
```

Comment:
where is toString defined for this?

Reply:
the default toString() was sufficient

Reply:
please define a string value for THUMBS_UP:

THUMBS_UP ("thumbs_up");

I understand the default is sufficient but please define it anyways so it doesn't keep the reader guessing. Especially since the value is important in the datastore and the front end.